mx.py

```diff
@@ -5358,8 +5358,8 @@ def is_release_from_tags(self, vcdir, prefix):
         :return: True if release
         :rtype: bool
         """
-        _release_version = self.release_version_from_tags(vcdir=vcdir, prefix=prefix)
-        return True if _release_version and re.match(r'^[0-9]+[0-9.]+$', _release_version) else False
+        _release_version = self.release_version_from_tags(vcdir=vcdir, prefix=prefix) #pylint: disable=assignment-from-no-return
```

Comment:
The called function doesn't have a return statement, since it's using `abort(...)` so this is a false positive